travis/scripts/02-generate-project.sh

```diff
@@ -16,14 +16,12 @@ if [ "$JHIPSTER" == "app-ng2-gateway-uaa" ]; then
     mkdir -p "$UAA_APP_FOLDER"
     cp -f "$JHIPSTER_SAMPLES"/uaa/.yo-rc.json "$UAA_APP_FOLDER"/
     cd "$UAA_APP_FOLDER"
-    yarn link generator-jhipster
```

Comment:
how does this link happen now? if yarn link doesnt work we should just use `npm link` here, they both can co exist without issues

Reply:
In fact, in Travis, I don't use `yarn link`, but use this instead: `yarn global add file:"$TRAVIS_BUILD_DIR"`, so it will install globally the version from this folder (the master version)

`yarn link` seems to be totally broken with Travis, and sometimes in my local too. When it happens, I need to clear some folders in `~/.config/yarn` and reinstall everything.

Reply:
Here the new line: https://github.com/jhipster/generator-jhipster/blob/master/.travis.yml#L72

Reply:
Just use `npm link` it works fine right

Reply:
the only command where Yarn shines is when you install stuff for everything else npm commands are perfectly fine as well